alicloud/resource_alicloud_polardb_endpoint_address.go

```diff
@@ -0,0 +1,249 @@
+package alicloud
+
+import (
+	"fmt"
+	"time"
+
+	"regexp"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/polardb"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+const polardbEndpointSuffixRegex = "\\.[a-zA-Z0-9\\-\\.]+\\.rds\\.aliyuncs\\.com"
+const polardbEndpointIdWithSuffixRegex = "^([a-zA-Z0-9\\-_]+:[a-zA-Z0-9\\-_]+:[a-zA-Z0-9\\-_]+)" + polardbEndpointSuffixRegex + "$"
+
+var polardbEndpointIdWithSuffixRegexp = regexp.MustCompile(polardbEndpointIdWithSuffixRegex)
+
+func resourceAlicloudPolarDBEndpointAddress() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudPolarDBEndpointAddressCreate,
+		Read:   resourceAlicloudPolarDBEndpointAddressRead,
+		Update: resourceAlicloudPolarDBEndpointAddressUpdate,
+		Delete: resourceAlicloudPolarDBEndpointAddressDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"db_cluster_id": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"db_endpoint_id": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"net_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"Public", "Private"}, false),
+				Default:      "Public",
+				ForceNew:     true,
+			},
+			"connection_prefix": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringLenBetween(1, 31),
+			},
+			"port": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"connection_string": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ip_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAlicloudPolarDBEndpointAddressCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+	clusterId := d.Get("db_cluster_id").(string)
+	dbEndpointId := d.Get("db_endpoint_id").(string)
+	netType := d.Get("net_type").(string)
+	prefix := d.Get("connection_prefix").(string)
+	if prefix == "" {
+		prefix = fmt.Sprintf("%stf", dbEndpointId)
```

Comment:
Please remove the default value and use system default value.

Reply:
> Please remove the default value and use system default value.

How can I do ? There are examples ?